prep-html-syntax/index.html

```diff
@@ -2,9 +2,7 @@
 <html>
 <h1 style="color: orange;">My favorite Star Wars Character is Baby Yoda</h1>
 <title>HTML Syntax</title>
-
 <body> <img src="https://static.highsnobiety.com/thumbor/4Yhd1KVQUaabmpbGCcu-uj-5yKU=/1600x1067/static.highsnobiety.com/wp-content/uploads/2019/12/11123117/baby-yoda-explained.jpg" alt="Baby Yoda" style="width: 500px; height: 600px;">
 <p style="color: blue;">I love Baby Yoda because he's so cute and cuddley.</p>
 </body>
-
 </html>
```

Comment:
Hi Diamond, looks like a couple extra lines were removed, but we didn't get the HTML rearranged into a separate `<head>` and `<body>`. That's ok, we can take a closer look during class 👍

Reply:
Hey, how do I resolve the conflict? It wont allow me to merge